src/System Application/App/Document Processor/src/PDFHelper.Codeunit.al

```diff
@@ -0,0 +1,33 @@
+
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.IO;
+
+using System.Utilities;
+
+/// <summary>
+/// Codeunit that provides helper functions for PDF processing.
+/// </summary>
+codeunit 3110 "PDF Helper"
+{
+    Access = Public;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    var
+        PDFHelperImpl: Codeunit "PDF Helper Impl";
+
+    /// <summary>
+    /// This procedure is used to convert a PDF file to an image.
+    /// </summary>
+    /// <param name="PdfInStream">Stream of the PDF file.</param>
+    /// <param name="ImageStream">Stream of the image file.</param>
+    /// <param name="ImageFormat">Image format to convert the PDF to.</param>
+    /// <param name="PageNumber">Page number to convert.</param>
+    procedure ConvertPdfToImage(PdfInStream: InStream; var ImageStream: InStream; ImageFormat: Enum "Image Format"; PageNumber: Integer)
```

Comment:
ConvertPageToImage, no need to repeat PDF and we are missing the Page to indicate it is one page.
Page number should go first.

We should get an overload that returns the first page I believe if that is the use case.
Do we need a method to return how many pages are in the document? Otherwise how could we loop and get all images?

Reply:
Why is it relevant? This pr adds simple function to convert a pdf page to image, as it is the current requirements. There is lots of work behind in the platform to enable it, and also license requirements we have to uphold. For now we are not going there.

Reply:
This function provides 1 page conversion.